scripts/update/mono.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 #Have I mentioned I hate mono?
 
-if [[ -f /install/.sonarr.lock ]] || [[ -f /install/.radarr.lock ]] || [[ -f /install/.lidarr.lock ]]; then
+if [[ -f /install/.sonarr.lock ]]; then
```

Comment:
This should be reverted, we don't know when #343 will be merged

Reply:
not fully reverted, radarr still doesn't need mono

i did add lidarr back and added a comment on 343 to remove lidarr from this script at that time.